doc/maintainers/team_process.md

```diff
@@ -180,6 +180,21 @@ now the most important thing is to direct users to a single destination.*
 
 ## Old issue closing policy
 
-We have yet to settle on a precise guide for closing out old issues, but, for
-now, if you happen across something that could easily be marked as resolved,
-please do so.
+A project like Stack can have a tendency to acquire issues over time, which can
+include:
```

Comment:
A (sortable, searchable) list of issues may not be the best inventory for all of these.

It would be helpful to see some idea of the current state of the project’s issues and PRs. I would strongly recommend a dedicated system to hold the tasks, separately to the issues.

That way, issues could be closed when nobody cares for them; rather than bumping an open issue, somebody interested could bump the closed issue, or perhaps vote on a task.

I wouldn't say this was an argument in favour of closing issues, but it would likely be a helpful part of _any_ solution.